routers/web/admin/admin.go

```diff
@@ -243,6 +243,7 @@ func Config(ctx *context.Context) {
 
 	ctx.Data["CustomConf"] = setting.CustomConf
 	ctx.Data["AppUrl"] = setting.AppURL
+	ctx.Data["GiteaRepoName"] = setting.Domain
```

Comment:
Why are you declaring both a context variable and a helper function? One of them ought to be enough and I'd lean to just using `Domain` which is set 1 line below this one, so we can remove both the helper and the variable.